launch/master_slaves_rgb8.launch

```diff
@@ -1,13 +1,13 @@
 <launch>
```

Comment:
I like the addition of `<?xml version="1.0"?>`; please add to this file too.

Reply:
done